pkgs/top-level/all-packages.nix

```diff
@@ -13775,33 +13775,7 @@ in
   vulkan-headers = callPackage ../development/libraries/vulkan-headers { };
   vulkan-loader = callPackage ../development/libraries/vulkan-loader { };
   vulkan-tools = callPackage ../tools/graphics/vulkan-tools { };
-  vulkan-validation-layers = callPackage ../development/tools/vulkan-validation-layers {
-    glslang = (glslang.override {
-      spirv-tools = spirv-tools.overrideAttrs (_: {
-        src = fetchFromGitHub {
-          owner = "KhronosGroup";
-          repo = "SPIRV-Tools";
-          rev = "26c1b8878315a7a5c188df45e0bc236bb222b698";
-          sha256 = "1q76vaqwxf4q2l4rd7j2p2jqgcqpys0m235drzx0drkn2qd50n1b";
-        };
-      });
-      spirv-headers = spirv-tools.overrideAttrs (_: {
-        src = fetchFromGitHub {
-          owner = "KhronosGroup";
-          repo = "SPIRV-Headers";
-          rev = "2434b89345a50c018c84f42a310b0fad4f3fd94f";
-          sha256 = "1m902q1alm0rbh69zlskkx4n453xijijp9mf3wzwphi2j36gygwm";
-        };
-      });
-    }).overrideAttrs (_: {
-      src = fetchFromGitHub {
-        owner = "KhronosGroup";
-        repo = "glslang";
-        rev = "e06c7e9a515b716c731bda13f507546f107775d1";
-        sha256 = "04y4dd1cqdkd4qffmhgmg3agf9j07ii2w38vpp4jw53ir818bqdq";
-      };
-    });
-  };
```

Comment:
What's the basis for moving away from upstream's known-good revisions here?

Reply:
The glslang already has overrides of spirv-tools and spirv-headers to known good versions. Seemed redundant.

Reply:
"known good" refers not just to the revisions of the dependencies themselves, but the package's requirements. For example, if a version earlier than the one identified by the package's known-good file is used, it supplied, it might be missing a critical interface, or worse, have silently incorrect behavior.

Reply:
That's a good point, could be silently failing. For the glslang package, it is using overrides to a "known-good" versions. I only removed the overrides from the `vulkan-validation-layers` package which used a completely different glslang version (and spirv-* packages by extension)

Reply:
Yes, vulkan-validation-layers calls for a specific commit of glslang (and by extension spirv-*) and we should respect that unless we have specific information that it is not necessary to do so.

Reply:
They do have a test suite, I try to get it pass. I think if it goes through the tests that we can consider it a "valid" glslang instance.

Reply:
added back the overrides, should be  back to a "valid" state now.